server/xpub-server/entities/manuscript/resolvers/uploadManuscript.test.js

```diff
@@ -122,11 +123,21 @@ describe('Manuscripts', () => {
         ),
         mimetype: 'application/pdf',
       }
+
+      jest.spyOn(scienceBeamApi, 'extractSemantics').mockRejectedValueOnce({
+        error: {
+          code: 'ETIMEDOUT',
+          connect: false,
+          message: 'timed out',
+        },
+      })
+
```

Comment:
We use replay to return a failed response from ScienceBeam. If we're not using it, we should remove the following https://github.com/elifesciences/elife-xpub/blob/2798152ee25c051d4a090403fda9ebf5faf97fcb/server/xpub-server/entities/manuscript/resolvers/uploadManuscript.test.js#L116

Reply:
Agreed